src/HotChocolate/ApolloFederation/src/ApolloFederation/Types/Directives/ComposeDirective.cs

```diff
@@ -24,10 +24,10 @@ namespace HotChocolate.ApolloFederation.Types;
 /// }
 /// </example>
 /// </summary>
-[Package(Federation25)]
+[Package(Federation21)]
```

Comment:
Is ComposeDirective already available in fed 2.1 I though it was introduced with 2.5

Reply:
OK ... just read your description :)

Reply:
yeah its been available for a while -> https://www.apollographql.com/docs/federation/federation-versions#v21

It is not really that useful ATM but we are hoping that we'll eventually get to it (maybe as part of the composite schemas WG)